README.md

```diff
@@ -107,25 +107,126 @@ Each of these can be run on an individual well using a plain `bash` script or as
 > [!NOTE]
 > All of the above commands are using `../../config.example.cfg` as the location of the config file, because of the layout of this repository. You can provide an **absolute path** to another location.
 
-## Demo
+### The configuration file
+
+The pipeline requires certain key parameters to be provided. For this we use a simple INI style plain-text file that can be parsed with the [`configparser` module](https://docs.python.org/3/library/configparser.html).  
+In the repository we provide an example configuration file, `config.example.cfg`. 
+
+| Parameter | Description | 
+|-----|-------------|
+| `[pipeline]` | |
+| plate_path   | path to where your raw images are | 
+| output_path   | path to where the csv files and zarr arrays will be saved   | 
+| well_targets   | name number of rows and columns (row1,col1&#124;row2,col2) of wells to analyze   | 
+| `[stitch_well]` | |
+| plane_size   |  size of image of one field, one z-slice and one channel   | 
+| overlap_x  | overlapping pixels between two adjacent fields   |
+overlap_y   | overlapping pixels between two adjacent fields   | 
+| `[cells_seg_well]` | |
+| output_path   | path to where the csv files will be saved   | 
+| stardist_path   | path to the trained model for nuclei segmentation  | 
 
-### Usage
-We have made an example dataset with one well data publicly available,
-the folder consists of images and .xml (metadata) file, and can be downloaded from: https://figshare.com/articles/dataset/cellos_data_zip/21992234 Once downloaded, it can be used as input example into the pipeline. The well row number=3 and column number=7. The image has 3 channels, channel1=EGFP, channel2=mCherry and channel3=brightfield. The expected results are under output folder.
 
+> [!NOTE]
+> The paths can be *relative* to the scripts, as is in the example provided here, which assumes the layout of the repository is fixed. Otherwise, the paths should be provided as **absolute paths**.
+
+## Usage
+
+We have made an example dataset with one well data publicly available. The well row number=3 and column number=7. The image has 3 channels, channel1=EGFP, channel2=mCherry and channel3=brightfield. 
+
+It can be downloaded from: https://figshare.com/articles/dataset/cellos_data_zip/21992234  
+
+On Linux, you can download it as follows:
+
+```bash
+wget https://figshare.com/ndownloader/files/39032216
+```
+
+> [!WARNING]
+> This is a ~11Gb zip file. 
+
+On Linux, it needs to be unziped using [7z](https://www.7-zip.org):
```

Comment:
I got errors using just `unzip`, perhaps due to the mac hidden dir inside or lack of extension? Google indicated that 7z is more robust and it worked.